src/DetailsView/Styles/detailsview.scss

```diff
@@ -172,94 +172,4 @@ body {
             }
         }
     }
-    .details-view-body-nav-content-layout {
```

Comment:
It looks like moving this into a module broke the ability to vertically scroll the left nav bar; `details-view-left-nav.scss` uses a `:global(.details-view-body-nav-content-layout)` to set height/max-height.